go/mysql/endtoend/query_test.go

```diff
@@ -264,35 +244,25 @@ func doTestWarnings(t *testing.T, disableClientDeprecateEOF bool) {
 	defer conn.Close()
 
 	result, err := conn.ExecuteFetch("create table a(id int, val int not null, primary key(id))", 0, false)
-	if err != nil {
-		t.Fatalf("create table failed: %v", err)
-	}
-	if result.RowsAffected != 0 {
-		t.Errorf("create table returned RowsAffected %v, was expecting 0", result.RowsAffected)
-	}
+	require.NoError(t, err, "create table failed: %v", err)
+	assert.Equal(t, uint64(0), result.RowsAffected, "create table returned RowsAffected %v, was expecting 0", result.RowsAffected)
 
 	// Disable strict mode
 	_, err = conn.ExecuteFetch("set session sql_mode=''", 0, false)
-	if err != nil {
-		t.Fatalf("disable strict mode failed: %v", err)
-	}
+	require.NoError(t, err, "disable strict mode failed: %v", err)
 
 	// Try a simple insert with a null value
 	result, warnings, err := conn.ExecuteFetchWithWarningCount("insert into a(id) values(10)", 1000, true)
-	if err != nil {
-		t.Fatalf("insert failed: %v", err)
-	}
+	require.NoError(t, err, "insert failed: %v", err)
+
 	if result.RowsAffected != 1 || len(result.Rows) != 0 {
 		t.Errorf("unexpected result for insert: %v", result)
 	}
```

Comment:
Can't we use this?
```
assert.True(t, (result.RowsAffected == 1 && len(result.Rows) == 0), "unexpected result for insert: %v", result)
```

Reply:
I would vote for separate assertions here, to make it easier to see what actually failed:

```go
assert.Equal(t, 1, result.RowsAffected, "unexpected rows affected by query; result: %v", result)
assert.GreaterThan(t, 0, len(result.Rows), "unexpected row count in query result: %v", result)
```

(also i never remember the ordering for the `GreaterThan` / `LessThan` assertions, so you should double check me on that 😆 )

Reply:
I agree with @ajm188 😄

Reply:
@mattlord I used a tool (https://github.com/arvind-murty/test-rewriter) to edit these files and it currently ignores the more complicated binary expressions (using `&&` or `||`), which is why this wasn't changed. This is also why some of the comments are in weird places.

I manually changed this instance, but I'm currently focused on getting through more packages to get more of the code base cleaned up.